governance/default.go

```diff
@@ -591,6 +596,9 @@ func (g *Governance) getGovernanceCache(num uint64) (map[string]interface{}, boo
 
 func (g *Governance) addGovernanceCache(num uint64, data GovernanceSet) {
```

Comment:
Calling `addIdxCache()` without locking might be dangerous. How about simply embedding the `addIdxCache()` to the call site?

Reply:
@jeonghonah The function was called by two functions before, but after fixing it now only one function is calling it. So as you suggested, it can be merged with the caller.
Thank you for your great feedback. =)